.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Generate and publish
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
```

Comment:
What's the purpose of fetch-tags true here? I'd assume that it shouldn't be necessary unless tags are explicitly interacted with, which I don't see in the following script.